pallets/crowdloan-rewards/Cargo.toml

```diff
@@ -29,6 +29,6 @@ std = [
     "serde",
     "log/std",
     "sp-std/std",
-    "sp-io/std"
-
+    "sp-io/std",
+    "pallet-balances/std"
```

Comment:
This shouldn't be necessary. We refer to the GLMR token through the associated type Config::RewardCurrency